src/bindings/cpp/include/keyset.hpp

```diff
@@ -603,6 +625,7 @@ inline ckdb::KeySet * KeySet::getKeySet () const
  * @see release()
  * @see getKeySet()
  */
+// TODO (kodebach): remove?
 inline void KeySet::setKeySet (ckdb::KeySet * k)
```

Comment:
I think the `setKeySet()` function should be removed as well. It is unused and IMO doesn't really make sense. There is no real point in resuing a `kdb::KeySet` object instead of calling the constructor to create a new `kdb::KeySet` from a `ckdb::KeySet *`.